inference-engine/tests/functional/plugin/gpu/shared_tests_instances/behavior/plugin/configuration_tests.cpp

```diff
@@ -8,10 +8,13 @@
 
 using namespace BehaviorTestsDefinitions;
 namespace {
-    const std::vector<InferenceEngine::Precision> netPrecisions = {
-            InferenceEngine::Precision::FP32,
-            InferenceEngine::Precision::FP16
-    };
+    INSTANTIATE_TEST_SUITE_P(
+            smoke_Basic,
+            DefaultConfigurationTest,
+            ::testing::Combine(
+                    ::testing::Values(CommonTestUtils::DEVICE_GPU),
+                    ::testing::Values(DefaultParameter{})),
```

Comment:
pls use some reasoble value, e.g.
                    ::testing::Values(DefaultParameter{GPU_CONFIG_KEY(PLUGIN_THROTTLE), InferenceEngine::Parameter{std::string{"0"}}})),

Reply:
fixed this  myself  for you :)